org.eclipse.xtext.ui.testing/src/org/eclipse/xtext/ui/testing/AbstractQuickfixTest.xtend

```diff
@@ -194,8 +198,7 @@ abstract class AbstractQuickfixTest extends AbstractEditorTest {
 		val in = new StringInputStream(Strings.emptyIfNull(model))
 		val uri = URI.createURI("") // creating an in-memory EMF Resource
 		
-		val rs = injector.getInstance(XtextResourceSet)
-		rs.setClasspathURIContext(AbstractQuickfixTest)
```

Comment:
this is imho a api/behaviour change. am not sure if we want that?

Reply:
@szarnekow what do you think?

Reply:
but i am not sure if the current impl makes any sense when project and rs dont fit together

Reply:
Since we now get the resourceSet associated with the project, it will already have the correct classpathURIContext. Looks about right to me.